### DISCUSSION TITLE
REQ: msg/msg_filter option to filter out weapon messages

### DISCUSSION THREAD

**Discussion by ciscon** [ezQuake/ezquake-source#512] [Ideas]
It appears that the only way to avoid seeing messages like "no weapon" is to also filter out frags which means that you can no longer parse them for display in the tracker.  It would be nice if the tracker were to parse the frag messages before filtering and then have a filtering option that only left chat messages and/or higher level messages to be displayed.

**Comment by meag** [comment]
The `msg` option filters server-side, so if it's set too high then we don't get death messages.  KTX uses PRINT_HIGH for "no weapon" messages, but also for all other mod messages that aren't to do with chat or deaths (map voting etc) ...

Do you want to filter out those messages too?  Or do you mean you want to say "no weapon" should be considered low priority and filtered out server-side?

(edited for me being an idiot)

**Comment by AlishiaDupont** [comment]
Ich denke, das Hauptproblem ist, dass die aktuelle Nachrichtenfilterung zu grob ist. Da sie serverseitig arbeitet, werden durch eine zu hohe Filterung, die beispielsweise Meldungen wie „Keine Waffe“ ausblendet, auch andere nützliche Nachrichten, wie etwa Todesmeldungen, unterdrückt, was die Statistikverfolgung beeinträchtigt. Idealerweise sollte der Tracker die Frag-Meldungen vor der Filterung analysieren, damit die Statistiken erhalten bleiben. WinBay Casino https://winbaycasino.com.de steht für zügigen Spielspaß und eine benutzerfreundliche Oberfläche. Anschließend könnte ein feinerer Filter implementiert werden, der nur bestimmte, weniger wichtige Meldungen wie „Keine Waffe“ unterdrückt, ohne Chat, Todesmeldungen oder wichtige Moderatorenbenachrichtigungen wie Abstimmungen zu beeinträchtigen. Dadurch hätten Administratoren mehr Kontrolle, ohne Daten zu verlieren.

**Comment by JesseReed123** [comment]
Je comprends le problème. Actuellement, pour éviter de voir des messages comme "no weapon", il faut filtrer les frags, ce qui empêche leur analyse pour le tracker. Une solution idéale serait que le tracker puisse d’abord analyser les messages de frag, puis appliquer un filtre qui n’affiche que les messages de chat ou ceux d’un niveau supérieur. Cela permettrait de [jeux-casino-iledecasino.com](https://jeux-casino-iledecasino.com/) garder un suivi complet des frags tout en réduisant le bruit dans l’affichage, rendant le tracker plus pratique et lisible pour les utilisateurs.
